pkg/app/piped/cmd/piped/piped.go

```diff
@@ -801,3 +834,30 @@ func loginToOCIRegistry(ctx context.Context, execPath, address, username, passwo
 	}
 	return nil
 }
+
+func stopCommandHandler(ctx context.Context, cmdLister commandstore.Lister, logger *zap.Logger) (bool, error) {
+	logger.Debug("fetch unhandled piped commands")
+
+	commands := cmdLister.ListPipedCommands()
+	if len(commands) == 0 {
+		return false, nil
+	}
+
+	var stopCmd *model.ReportableCommand
+	for _, command := range commands {
+		if command.IsRestartPipedCmd() {
+			stopCmd = &command
+			break
+		}
+	}
+
+	if stopCmd == nil {
+		return false, nil
+	}
+
+	if err := stopCmd.Report(ctx, model.CommandStatus_COMMAND_SUCCEEDED, nil, []byte(stopCmd.Id)); err != nil {
+		return false, fmt.Errorf("failed to report command %s: %w", stopCmd.Id, err)
+	}
```

Comment:
@khanhtc1202 User could send multiple commands from web console, so how about marking all of them as success before exiting?

Reply:
For sure, updated 👍